examples/step-55/step-55.cc

```diff
@@ -193,18 +193,20 @@ namespace Step55
 
     virtual void vector_value(const Point<dim> &p,
                               Vector<double> &  value) const override;
+
+  private:
+    static constexpr double pi  = numbers::PI;
+    static constexpr double pi2 = numbers::PI * numbers::PI;
   };
 
 
   template <int dim>
   void RightHandSide<dim>::vector_value(const Point<dim> &p,
                                         Vector<double> &  values) const
   {
-    const double R_x = p[0];
-    const double R_y = p[1];
+    const double &R_x = p[0];
+    const double &R_y = p[1];
 
-    const double pi  = numbers::PI;
-    const double pi2 = pi * pi;
     values[0] =
       -1.0L / 2.0L * (-2 * sqrt(25.0 + 4 * pi2) + 10.0) *
         exp(R_x * (-2 * sqrt(25.0 + 4 * pi2) + 10.0)) -
```

Comment:
just use `numbers::pi * numbers::pi` here.

While there, can you also `std::`-qualify the calls to `sqrt`?

Reply:
> just use `numbers::pi * numbers::pi` here.

Okay, I will revert the changes. Or we could use the `PI_SQ` idea I mentioned above.

> While there, can you also `std::`-qualify the calls to `sqrt`?

Will do!

Reply:
> just use `numbers::pi * numbers::pi` here.

I reverted the changes and turned the variables into `constexpr`.

> While there, can you also std::-qualify the calls to sqrt?

For this, I've added a `using namespace std`. I think this increases readability of the code.

Reply:
That would be the only occurrence of `using namespace std` in the entire `examples/` directory. Let's do things the same way they are done everywhere else, if you don't mind.

Reply:
I think the motivation was that this block of code can be copy&pasted from ``reference.py`` and that is why variables are named awkwardly and there is no ``std::`` to be found. I am not opposed to the updates proposed here, though.